python_bindings/src/PyExpr.cpp

```diff
@@ -8,6 +8,15 @@ namespace Halide {
 namespace PythonBindings {
 
 void define_expr(py::module &m) {
+    auto to_bool = [](const Expr &e) -> bool {
+        std::ostringstream o;
+        o << e;
+        throw py::value_error("The halide.Expr (" + o.str() + ") cannot be converted to a bool. "
+            "If you encountered this error using the 'and'/'or' keywords, "
```

Comment:
Somehow "you" in an error messages reads wrong to me. Perhaps: "If this error occurs using the 'and'/'or' keywords, consider using the '&'/'|' operators instead."